src/assets/wise5/services/cRaterService.ts

```diff
@@ -301,4 +304,49 @@ export class CRaterService {
         return isAvailable;
       });
   }
+
+  public getDataFromResponse(response: any): CRaterResponse {
+    const data: any = {};
```

Comment:
I think this function needs to return an actual CRaterResponse instance. Otherwise, code downstream won't be able to call its functions, like ```response.isNonScorable()``` in DialogGuidanceFeedbackRuleEvaluator.
```suggestion
    const data: CRaterResponse = new CRaterResponse();
```